src/components/tools/text-tool.tsx

```diff
@@ -81,6 +93,24 @@ export default class TextToolComponent extends BaseComponent<IProps, IState> {
   private disposers: IReactionDisposer[];
   private prevText: any;
 
+  private plugins: Plugin[] = [
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+b", type: "bold" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+i", type: "italic" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+u", type: "underlined" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+.", type: "superscript" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+,", type: "subscript" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.block, key: "ctrl+alt+0", type: "deleted" }),
```

Comment:
I hope/assume that these keyboard shortcuts translate `ctrl` to `cmd` on Mac? Also, I don't particularly care what the keyboard shortcuts are, but is `ctrl-~` i.e. `` ctrl-shift-` `` available for delete? `ctrl-alt-0` seems particularly hard to remember.

Reply:
Actually, they do not; although, we did have an earlier version that did. To effect this change, in theory, the `key` member could be changed to `"mod+u"` for example where the `mod` modifier is swapped between the Mac-% command key and the Windows Ctrl key.

A say "in theory" because I didn't have a chance to test it out on a Windows box, or a Chromebook.

Reply:
Whatever we check in should work on Macs, so if this doesn't I'd say that's a required change.

Reply:
After looking at the documentation for `is-hotkey` I've gone and replaced the instances of `ctrl` with `mod`. This is working on a Chromebook, I'll update here if it's working on a MBP and a Windows laptop after.

Reply:
I've verified this works on a Chromebook and a MacBook Pro! I'm pretty certain this will work on a Windows machine too with all things considered there.

Reply:
With this latest change, on a Mac running Chrome, `CMD-b`, `CMD-i`, `CMD-u`, and `CMD-,`, work, as desired for bold, italic, underline, and subscript.

`CMD-.` (for superscript) and the combination "mod+alt+0" for deleted-text do not work. In the case of `CMD-.`, Chrome maps that to an "open Chrome apps" widget.

For now, I'd suggest, comment out the `delete` action, and change the hot-key mapping for superscript from `mod+.` to `mod+shift+,` (note the comma). This hot-key combo does seem to be available and is somewhat more logical. As in, `CMD-,` points down for a subscript and shifted, `CMD-SHIFT-,` is it's compliment.

Reply:
That's really odd, I also did this test out on a MBP running Chrome and did not run into the issue you've described with the superscript and subscript, but regardless this is a likely good change anyways. Commented out the delete action!